tests/test_ods_output_diff_jp.py

```diff
@@ -0,0 +1,345 @@
+# Copyright 2022 Neal Chambers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import unittest
+
+# from datetime import date
+from pathlib import Path
+
+from abstract_test_ods_output_diff import AbstractTestODSOutputDiff, OutputPlugins
+
+ROOT_PATH: Path = Path(os.path.dirname(__file__)).parent.absolute()
+
+
+class TestODSOutputDiff(AbstractTestODSOutputDiff):  # pylint: disable=too-many-public-methods
+
+    output_dir: Path
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.output_dir = ROOT_PATH / Path("output") / Path(cls.__module__)
+
+        shutil.rmtree(cls.output_dir, ignore_errors=True)
+
+        AbstractTestODSOutputDiff._generate(
+            cls.output_dir, test_name="crypto_example", config="crypto_example", method="fifo", country="jp", generation_language="en"
+        )
+        # AbstractTestODSOutputDiff._generate(cls.output_dir, test_name="test_data", config="test_data", method="fifo")
```

Comment:
Remember to remove commented code when you're ready to merge.

Reply:
No worries, it's all good: I just always review everything as if it was final, which is a bit pedantic, but at least it minimizes the risk of leaving something out.